lib/common/session_model.dart

```diff
@@ -247,11 +249,17 @@ class SessionModel extends Model {
   }
 
   Devices devicesFromJson(dynamic item) {
-    final items = item as List<dynamic>;
-    if (items.length == 0) return Devices.create();
-
-    final res = jsonEncode(items);
-    return Devices.create()..mergeFromProto3Json(jsonDecode(res));
+    final devices = <Device>[];
+    for (final element in item) {
```

Comment:
Here I was getting an error that says  Json is expected, I ran a loop and passed json to protobuf

Reply:
I made my user Pro via the panel, but I also have no devices so I'm not sure I have any way of testing this. Did you confirm you see the device list when you navigate to the Pro account screen?

![image](https://github.com/getlantern/lantern-client/assets/6346213/1ab0189a-e32f-4071-9c42-d2bdc76e510b)

Reply:
Yes,I did, I see my devices on screen